test/packages/good/data_stream/foo/fields/external-fields.yml

```diff
@@ -3,4 +3,7 @@
   description: Event family
   fields:
     - name: category
-      external: ecs
\ No newline at end of file
+      external: ecs
+    - name: module
+      type: constant_keyword
```

Comment:
I see that this is an inconsistency comparing to the original source - ECS repository, which defines it as `keyword`:

```
    - name: module
      level: core
      type: keyword
      ignore_above: 1024
      description: 'Name of the module this data is coming from.
        If your monitoring agent supports the concept of modules or plugins to process
        events of a given source (e.g. Apache logs), `event.module` should contain
        the name of this module.'
```

Do we plan to adjust the ECS schema too (`event.module`)?

This brings me to the final question - currently the import logic in elastic-package imports fields as they are defined. In this case (`event.module`) the definition would be altered with local properties (`value`, `type`). I'm making sure this is expected behavior.

Reply:
> Do we plan to adjust the ECS schema too (event.module)?

AFAIK there are plans at the moment to change the definition of `event.module`. But there have been some talks of loosing the specification to allow implementors to use either keyword or constant_keyword.

// fyi: @elastic/ecs 

> In this case (event.module) the definition would be altered with local properties (value, type). I'm making sure this is expected behavior.

I was expecting that packages would be required to declare their full definition of `event.module` if they make it a `constant_keyword`.

Reply:
If you prefer you can leave overriding for further improvement (not to block this PR).

Reply:
> AFAIK there are plans at the moment to change the definition of event.module.

I've heard some discussion, but I'm not aware of a decision being made yet.

I suggested allowing `keyword` and `constant_keyword` to be interchangeable and avoid requiring a specific indexing strategy for all ECS users who wish to use `event.module`.